typescript/README.md

```diff
@@ -0,0 +1,94 @@
+# @cooklang/cooklang-ts
+
+A TypeScript/JavaScript library for parsing CookLang recipes, built with Rust and WebAssembly.
+
+> [!NOTE]  
+> Package is under development. What exists here may not be published to NPM yet.
+
+## Prerequisites
```

Comment:
Presuming this README becomes the content published with this package, we should likely move this whole section. It is only applicable to someone trying to set up the dev environment and is not targeted at a consumer. Considering the amount of context we will likely need to include, it might all be best in a `CONTRIBUTING.md` that we point at from this `README.md`.

Reply:
Good point! I'll fix this and the other comment you had in this file